retry/retry.go

```diff
@@ -193,7 +194,7 @@ func Retryable(err error) bool {
 
 	var mye *mysql.MySQLError
 	var pqe *pq.Error
-	if errors.As(err, &mye) || errors.As(err, &pqe) {
+	if errors.As(err, &mye) || errors.As(err, &pqe) || strings.HasPrefix(err.Error(), "pq: ") {
```

Comment:
Could you please add a bit more information here? Does this case necessary belong to this `if`?

Personally, I am a bit puzzled by this change.

Reply:
> This definitely isn't how best-practice error handling in Go should look like,

Not so loud, please! 🙈 https://github.com/Icinga/icingadb/blob/v1.2.0/pkg/icingaredis/utils.go#L96

Reply:
That's just returning an error, not error handling. Should we have the need to handle that error specifically, that should be changed to its own type rather than checking for that string, that's all.

Reply:
> No, I didn't found any: https://github.com/lib/pq/issues?q=is%3Aissue+is%3Aopen+errorf

Now there's one: https://github.com/lib/pq/issues/1169